rcl/src/rcl/arguments.c

```diff
@@ -108,14 +108,12 @@ _rcl_parse_remap_rule(
   if (len_node_name) {
     int validation_result;
     size_t invalid_index;
-    if (
-      RMW_RET_OK != rmw_validate_node_name_with_size(arg, len_node_name, &validation_result,
-      &invalid_index))
-    {
+    rmw_ret_t rmw_ret = rmw_validate_node_name_with_size(arg, len_node_name, &validation_result,
```

Comment:
Nitpick: I would suggest to wrap after the opening parenthesis so that all arguments are on the indented next line.